### ISSUE TITLE
Logs on Centos 6.7 contain invalid characters around timestamps

### ISSUE THREAD

**Issue by rjemanuele** [virgo-agent-toolkit/rackspace-monitoring-agent#918]
Example:

[logcorrupt.txt](https://github.com/virgo-agent-toolkit/rackspace-monitoring-agent/files/81674/logcorrupt.txt)

**Comment by jjbuchan** [comment]
This also appears to be an issue on CentOS 7 and debian.  Confirmed to happen in agent v2.4.2

```
 4:04:49 PM 2016 INF: Check (agent.load_average) -> agent.load_average (details=,id="chcFJdTh2A",period=60) scheduled for 60s<D8><C8>)<B6>A^@^@^@^@<C8>7
<B6>A^@^@^@^@X<E2><F6>A^@^@^@^@^@^@^@^@^@^@^@^@ Check (agent.load_average) -> agent.load_average (details=,id="chcFJdTh2A",period=60) scheduled for 60s
```

**Comment by rphillips** [comment]
Fixed in 2.4.3
